### ISSUE TITLE
Accessibility Issue in Media content pagination

### ISSUE THREAD

**Issue by ndouglas** [department-of-veterans-affairs/va.gov-cms#12912]
## Description
Seeing this Axe error:

![Screenshot 2023-03-09 at 2 45 05 PM](https://user-images.githubusercontent.com/1318579/224141121-22586f77-24b7-414a-bf9f-2d7f0feaccc7.png)

for this item:

![Screenshot 2023-03-09 at 2 44 36 PM](https://user-images.githubusercontent.com/1318579/224141145-b9f23d0c-0844-4021-96a2-b1e31688d56c.png)

on `/admin/content/media`.

## Acceptance Criteria
- [ ] Testable_Outcome_X
- [ ] Testable_Outcome_Y
- [ ] Testable_Outcome_Z
- [ ] Requires design review

### Team
Please check the team(s) that will do this work.

- [x] `CMS Team`
- [ ] `Public Websites`
- [ ] `Facilities`
- [ ] `User support`

**Comment by laflannery** [comment]
I looked into this and this is done pretty much everywhere as a very established pattern (Drupal, VA, USWDS). I have a proposed solution but I'm running it by the other a11y folks just because I wanted some other eyes.

@ndouglas For now, if this is blocking you or others from completing tests or merging code I do not have an issue with you writing an ignore in for this since I have this documented and as it's upstream might take a bit to get resolved. If you do, can you note it in here so we can remember to remove it later?

**Comment by ndouglas** [comment]
Yeah, that works for me.  Thank you!

**Comment by laflannery** [comment]
Core issue: https://www.drupal.org/project/drupal/issues/3348552

**Comment by productmike** [comment]
@tonytaylor @edmund-dunn If able to a deeper look at the patch and update story point as necessary. F/up 7/5

**Comment by productmike** [comment]
@edmund-dunn @tonytaylor 
Can y'all include this in your pre-finement discussions prior to next week's refinement session?

cc: @laflannery @BerniXiongA6

**Comment by ndouglas** [comment]
Edmund suggests this should be prioritized post-D10.

**Comment by srancour** [comment]
Still an issue and should be addressed.
